tasks/push-rpm-data-to-pyxis/README.md

```diff
@@ -13,6 +13,10 @@ all repository_id strings found in rpm purl strings in the sboms.
 | server          | The server type to use. Options are 'production','production-internal,'stage-internal' and 'stage'. | Yes      | production    |
 | concurrentLimit | The maximum number of images to be processed at once                                                | Yes      | 4             |
 
+## Changes in 1.1.1
```

Comment:
The yaml has a new result `sbomPath` that isn't even mentioned here. Also, a new result is not just a small bugfix. I would go to 1.2.0

Reply:
Updated